api/src/pcapi/static/backofficev3/js/addons/pc-field-list.js

```diff
@@ -66,14 +67,14 @@ class PcFieldList extends PcAddOn {
   }
 
   #generateNextElementNumber = ($ul) => {
-    const entriesCount = $ul.dataset
+    const entriesCount = $ul.dataset.entriesCount
```

Comment:
Destructuring plus légé :

```suggestion
    const { entriesCount } = $ul.dataset
```

Reply:
Les deux lignes sont équivalentes ?